__tests__/app.test.js

```diff
@@ -381,7 +382,39 @@ describe('GET: /api/users', () => {
                 expect(body.msg).toBe("Endpoint doesn't exist.")
 
             })
-        })
-    
+    })
+})
+
+describe('GET /api/articles (topic query)', () => {
+    test('status:200, responds with all articles filtered by topic', () => {
+        return request(app)
+            .get('/api/articles?topic=mitch')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles).toHaveLength(12)
+                body.articles.forEach((article) => {
+                    expect(article).toMatchObject(
+                        {
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            article_id: expect.any(Number),
+                            topic: 'mitch',
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            article_img_url: expect.any(String),
+                            comment_count: expect.any(String)
+                        })
+                })
+            })
+    })
+
+    test('status:200, responds with an empty array if topic does not exist', () => {
```

Comment:
Ideally this would be the case if the topic DOES exists, but has no articles yet. For instance 
Status 200, valid topic query, but has no articles responds with an empty array of articles, e.g. ?topic=paper.
Then we should have a test for
Status 404, non-existent topic query, e.g. ?topic=bananas.